pkgs/top-level/python-packages.nix

```diff
@@ -25935,6 +25935,8 @@ in {
     };
   };
 
+  termdown = callPackage ../applications/misc/termdown {};
```

Comment:
Python applications are not called via `python-packages.nix` but directly from `all-packages.nix`.